### ISSUE TITLE
Request for onDelete props when using the jsonforms schema of type='array.

### ISSUE THREAD

**Issue by sunilb0575** [eclipsesource/jsonforms#1286]
**Is your feature request related to a problem? Please describe.**
Request for onDelete props when using the jsonforms schema of type='array.
This would be helpful in updating the deleted object. 

**Describe the solution you'd like**

Array before deleting:
```
case_workers_primary:
[{id: 10, user_id: 4, start_date: "2019-02-28"}, {id: 11, user_id: 3, start_date: "2019-03-21"}]
```
and on deleting the second object in the array, I need onDelete or onClick props so that I can change the second object to ```{id: 11, destroyed: true}```

then the array would look like this 

```
[{id: 10, user_id: 4, start_date: "2019-02-28"}, {id: 11, destroyed: true}
```

**Describe for which setup you like to have the improvement**
Framework: [e.g. react]
RendererSet: [e.g. material]

**Comment by edgarmueller** [comment]
Thanks for the report. As you already outlined I think this use case can be generalized such that it is covered by #1271 as well, but we'll first need to think about how the API for this can look like. We'll keep you posted.

**Comment by sunilb0575** [comment]
thank you @edgarmueller for the response.

**Comment by Lily418** [comment]
Is this now closed by #1448?

**Comment by sdirix** [comment]
This issue will be closed as it has been open for a long time with no activity. Feel free to reopen this issue if needed.
